src/schemas/21P-530/schema.js

```diff
@@ -40,17 +44,25 @@ let schema = {
           type: 'string'
         }
       }
-    }
+    },
+    claimantAddress: schemaHelpers.getDefinition('address'),
+    claimantEmail: {
+      type: 'string',
+      format: 'email'
+    },
+    claimantPhone: schemaHelpers.getDefinition('usaPhone'),
   }
 };
 
 [
+  ['address'],
   ['fullName', 'claimantFullName'],
   ['fullName', 'veteranFullName'],
   ['ssn', 'veteranSocialSecurityNumber'],
   ['vaFileNumber'],
   ['date', 'burialDate'],
   ['date', 'deathDate'],
+  ['usaPhone'],
```

Comment:
this isn't needed if it's already been added manually. what `schemaHelpers.addDefinitionToSchema` does is try to import the definition from definitions.js if it doesn't already exist. then first argument is the definition name and the 2nd argument is the name of the field to put the definition in